spring-observability-core/src/main/java/org/springframework/observability/tracing/listener/TracingTagFilter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.observability.tracing.listener;
+
+import org.springframework.observability.event.tag.Tag;
+import org.springframework.observability.tracing.Span;
+
+class TracingTagFilter {
+
+	void tagSpan(Span span, Iterable<Tag> tags) {
+		tags.forEach(tag -> {
+			if (tag.getKey().equalsIgnoreCase("error") && tag.getValue().equalsIgnoreCase("none")) {
```

Comment:
Is this a case we need to handle?
There should not be an `error=none` except the users set it explicitly.
The micrometer listener just tag the timer, it should not touch the recording.

Reply:
That's the default behaviour for Spring Boot